browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -388,8 +388,8 @@ L.CalcTileLayer = L.CanvasTileLayer.extend({
 
 		// Document container size is up to date as of now.
 		const documentContainerSize = this._getDocumentContainerSize();
-		documentContainerSize[0] = Math.round(app.dpiScale * documentContainerSize[0]);
-		documentContainerSize[1] = Math.round(app.dpiScale * documentContainerSize[1]);
+		documentContainerSize[0] *= app.dpiScale;
```

Comment:
no rounding needed?

Reply:
It doesn't make a difference here. Resulting values are rounded.